conversation.go

```diff
@@ -100,6 +107,41 @@ func (api *Client) GetUsersInConversationContext(ctx context.Context, params *Ge
 	return response.Members, response.ResponseMetaData.NextCursor, nil
 }
 
+// GetConversationsForUser returns the list conversations for a given user
+func (api *Client) GetConversationsForUser(params *GetConversationsForUserParameters) (channels []Channel, nextCursor string, err error) {
+	return api.GetConversationsForUserContext(context.Background(), params)
+}
+
+// GetConversationsForUserContext returns the list conversations for a given user with a custom context
+func (api *Client) GetConversationsForUserContext(ctx context.Context, params *GetConversationsForUserParameters) (channels []Channel, nextCursor string, err error) {
```

Comment:
it'd be better if you returned the entire GetConversationsForUserParameters data for use. slightly easier to use as an API.

Reply:
It seems like the rest of the conversation.go method have the same api that lets delegates that work to the caller. I didn't want to introduce a new method that functioned differently. I can definitely add another higher level method for the caller to return all data if you'd like.

Reply:
fair point, can always change it later.